packages/terra-arrange/tests/nightwatch/arrange-spec.js

```diff
@@ -5,30 +5,130 @@ module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous
   'Displays an arrange with fitStart, fill, and fitEnd content aligned together': (browser) => {
```

Comment:
It might be better to break these tests out into smaller test steps which defines the difference between default, center, bottom, etc.

Reply:
@emilyrohrbough Can you give an example of how you would define `center`? Just trying to figure out how to address this.

Reply:
Talked in Teams.

How's this: https://github.com/cerner/terra-core/pull/830/commits/ff1749c9b82fa74651a94a7cac9efb37ed1b24fe ?